src/state.py

```diff
@@ -89,8 +117,10 @@ def from_charm(cls, charm: ops.CharmBase) -> "IntegratorInformation":
         Returns:
             IntegratorInformation: Instance of the state component.
         """
-        backend_addresses = typing.cast(str, charm.config.get("backend-addresses"))
-        backend_ports = typing.cast(str, charm.config.get("backend-ports"))
+        backend_addresses = cast(str, charm.config.get("backend-addresses"))
+        backend_ports = cast(str, charm.config.get("backend-ports"))
+        paths = cast(str, charm.config.get("paths"))
```

Comment:
I think you don't have a test for the happy path. I'd consider extending `state.py:test_get_integrator_information`